### DISCUSSION TITLE
What's the meaning of `Port an existing TextMate theme to VS Code.`?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#261] [Extension Development QnA] [answered]
When develop a theme, the theme has a function:

```
Port an existing TextMate theme to VS Code.
```

what's the meaning of it?

**Comment by alefragnani** [comment, marked as answer]
Hi @liaodalin19903 ,

This is related to syntax colors, as described in https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#theming and https://code.visualstudio.com/api/extension-guides/color-theme#syntax-colors.

Since TextMate themes allows you to define colors to tokens, you can reuse it/port it to VS Code as well

Hope this helps
